test/controllers/responsiblepeople/SoleProprietorOfAnotherBusinessControllerSpec.scala

```diff
@@ -209,9 +210,9 @@ class SoleProprietorOfAnotherBusinessControllerSpec extends AmlsSpec with Mockit
         "display page Not Found" when {
           "neither soleProprietorOfAnotherBusiness nor name is set" in new Fixture {
 
-            when(mockDataCacheConnector.fetch[Seq[ResponsiblePerson]](any())(any(), any(), any()))
+            when(mockDataCacheConnector.fetch[Seq[ResponsiblePerson]](any(), any())(any(), any()))
               .thenReturn(Future.successful(Some(Seq(ResponsiblePerson()))))
-            when(controller.statusService.isPreSubmission(any(), any(), any()))
+            when(controller.statusService.isPreSubmission(Some(any()), any(), any())(any(), any()))
```

Comment:
thing to consider in the future (no changed required now): instead of Some(any()) what looks a bit weird I think, you could explicitly say what kind of type you're matching against using:
Some(anyString()) or Some(any[String]) or any[Option[String]]

but of course in the form you've used it works as well.

Reply:
Thats a good shout, ill make sure to do this in the future